nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/dao/impl/StandardLabelDAO.java

```diff
@@ -99,7 +101,9 @@ public Label updateLabel(LabelDTO labelDTO) {
             label.setPosition(new Position(labelDTO.getPosition().getX(), labelDTO.getPosition().getY()));
         }
         if (labelDTO.getStyle() != null) {
-            label.setStyle(labelDTO.getStyle());
+            final Map<String, String> updatedStyles = new HashMap<String, String>(label.getStyle());
```

Comment:
@pvillard31 minor detail, re-declaring the types is not needed:
```suggestion
            final Map<String, String> updatedStyles = new HashMap<>(label.getStyle());
```